README.md

```diff
@@ -72,19 +86,30 @@ Write one Express route for each of the four HTTP methods.
 Then, make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route.
 
 ```js
-var express = require("express");
-var app = express();
-
-// Your code starts here...
+const express = require("express")
+const app = express()
 
+app.get("/", (req, res)=>{
+  console.log("GET")
+})
+app.post("/puppies", (req, res)=>{
+  console.log("POST")
+})
+app.post("/puppies/:id", (req, res)=>{
+  console.log("PUT")
+}
+app.delete("/puppies/:id", (req, res)=>{
+  console.log("DELETE THE PU- this is pretty morbid.")
+})
 ```
 
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-Your answer...
+Express is a NODE framework whereas Rails is for Ruby.
+Express is highly modular, expecting most of its functionality to come from modules imported in, such as handlebars, body-parser, among other things.  It's a lightweight, un-opinionated framework that allows a high degree of variability and customization whereas Rails expects convention over configuration.  Rails is highly opinionated, that is it has very specific guidelines as to how to write things, so it ends up being very rigid (rails is an apt name).  It's easy to use, but not as forgiving on novel thought.
 ```
 
```

Comment:
👌 

The rigidity of Rails is mostly in its implementation of RESTful conventions. Overall, it is very flexible.